jerry-core/parser/js/js-parser-expr.c

```diff
@@ -791,11 +791,21 @@ parser_parse_object_literal (parser_context_t *context_p) /**< context */
         uint32_t status_flags;
         cbc_ext_opcode_t opcode;
         uint16_t literal_index, function_literal_index;
+        bool is_getter = context_p->token.type == LEXER_PROPERTY_GETTER;
```

Comment:
There is no need to save this value, `lexer_check_next_character` does not change it.